src/components/user-list-test.html

```diff
@@ -0,0 +1,49 @@
+<link rel="import" href="../../node_modules/@banno/polymer/polymer.js">
+<link rel="import" href="../../node_modules/@banno/polymer/lib/elements/dom-repeat.js">
+<link rel="import" href="./user-profile.html">
+
+<dom-module id="user-list">
+  <style>
+  </style>
+  <template>
+    <h2> User list: </h2>
+    <dom-repeat items="{{users}}">
+      <template>
+        <div>
+          <user-profile 
+            id="{{item.id}}" 
+            first="{{item.first}}" 
+            last="{{item.last}}" 
+            phone="{{item.phone}}"
+            email="{{item.email}}">
+          </user-profile>
+        </div>
+      </template>
+    </dom-repeat>
+  </template>
+
+  <script type="module">
+    import { Element as PolymerElement } from '@banno/polymer/polymer-element.js';
+    import { LocalStorage } from '../api/localStorage.js';
+
+    const LocalUsers = new LocalStorage();
+
+    class UserList extends PolymerElement {
+      static get is() { return 'user-list'; }
+      static get properties() {
+        return {
+          users: {
+            value() {
+              return [
```

Comment:
Don't need this since you're populating and passing users into the component. Just need to declare it's type.
```suggestion
             users: Array,
```